tests/e2e/server/routes.auth.mjs

```diff
@@ -225,6 +225,13 @@ export default function (app) {
   });
 
   app.get(authPaths.authorize, wait, async (req, res) => {
+    // Detect if Central Login to enforce ACR value presence
+    if (
+      req.query.client_id === 'CentralLoginOAuthClient' &&
+      req.query.acr_values !== 'SpecificTree'
+    ) {
```

Comment:
Here's where I'm isolating just the Central Login for testing ACR values. All other OAuth test automation uses a different client ID.

Reply:
👍